testing_and_setup/compass/README_ocean.md

```diff
@@ -6,21 +6,25 @@ To set up and run ocean test cases from COMPASS, you will need a conda
 environment.  First, install Miniconda3 (if miniconda is not already
 installed), then create a new conda environment as follows:
 ``` bash
-conda create -n compass_py3.7 -c conda-forge -c xylar python=3.7 \
-    geometric_features=0.1.4 mpas_tools=0.0.6 jigsaw=0.9.11 jigsawpy=0.0.2 \
-    metis pyflann scikit-image basemap pyamg ffmpeg
+conda create -n compass_0.1.0 -c conda-forge -c e3sm python=3.7 compass=0.1.0
```

Comment:
This is now out of date. I'll just use the second commit, as the first commit updated the conda version.

Reply:
Correct.  This is now taken care of in the `compass` metapackage.